### DISCUSSION TITLE
When changing a Repository from Private to Public all images uploaded become unaccesible

### DISCUSSION THREAD

**Discussion by CondensedChaos** [community/community#61303] [Repositories]
### Select Topic Area

Bug

### Body

Hello.

I made a great deal of work on the readme.md of Repository while being private, I just wanted to make things perfect and waited when it was ready to make it public but when I change the visibility of the repository from private to public I lose access to all image in the readme.md file.
Changing the repository from public to private again will show all the images again so the images are there.

When private the images URL are: in the format https://private-user-images.githubusercontent.com/[some code]/[filename].png?jwt=[some other code]
when public they are in the format https://user-images.githubusercontent.com/[some code]/[file name].png

This became a huge bummer for me considering the huge time I spent and it pushes the release date further along if I have to reupload all the images again in their correct place, as I was ready to release my project.

Is anybody aware of this issue and if there's some way to get a fix?

Thanks

**Comment by ClassyJuhi** [comment]
Problem: Changing your private repository to public caused image URLs to change, making images in readme.md inaccessible.

Cause: GitHub uses different URLs for images in private and public repositories.

Solution:

a. Open the readme.md file in your repository.

b. Update the image URLs from the private format:
Change https://private-user-images.githubusercontent.com/...
To https://user-images.githubusercontent.com/...

c. Save the changes and commit them to your repository.

Result: The images should now be accessible and displayed correctly in your readme.md file in the public repository.

By following these steps, you won't need to re-upload the images, and your repository will be ready for release with the images properly visible to the public.
